src/styles/blocks/intro.scss

```diff
@@ -0,0 +1,105 @@
+.intro {
+  --w: 100%;
```

Comment:
use scss variables

Reply:
Fixed everything except this coz slider doesn't work with scss variables :(